doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -216,190 +216,200 @@ of the following form:
 Starting the OSD
 ----------------
 
-After you add an OSD to Ceph, the OSD is in your configuration. However,
-it is not yet running. The OSD is ``down`` and ``in``. You must start
-your new OSD before it can begin receiving data. You may use
-``service ceph`` from your admin host or start the OSD from its host
-machine:
+After an OSD is added to Ceph, the OSD is in the configuration. However, until
+it is started, the OSD is considered ``down`` and ``in``. The OSD is not
+running and will be unable to receive data. To start an OSD, either run
+``service ceph`` from your admin host or run a command of the following form to
+start the OSD from its host machine:
 
    .. prompt:: bash $
 
       sudo systemctl start ceph-osd@{osd-num}
 
+After the OSD is started, it is considered ``up`` and ``in``.
 
-Once you start your OSD, it is ``up`` and ``in``.
+Observing the Data Migration
+----------------------------
 
-
-Observe the Data Migration
---------------------------
-
-Once you have added your new OSD to the CRUSH map, Ceph  will begin rebalancing
-the server by migrating placement groups to your new OSD. You can observe this
-process with  the `ceph`_ tool. :
+After the new OSD has been added to the CRUSH map, Ceph begins rebalancing the
+server by migrating placement groups (PGs) to the new OSD. To observe this
+process by using the `ceph`_ tool, run the following command:
 
    .. prompt:: bash $
 
       ceph -w
 
-You should see the placement group states change from ``active+clean`` to
-``active, some degraded objects``, and finally ``active+clean`` when migration
-completes. (Control-c to exit.)
+The PG states will first change from ``active+clean`` to ``active, some
+degraded objects`` and then return to ``active+clean`` when migration
+completes. When you are finished observing, press Ctrl-C to exit.
 
 .. _Add/Move an OSD: ../crush-map#addosd
 .. _ceph: ../monitoring
 
 
-
 Removing OSDs (Manual)
 ======================
 
-When you want to reduce the size of a cluster or replace hardware, you may
-remove an OSD at runtime. With Ceph, an OSD is generally one Ceph ``ceph-osd``
-daemon for one storage drive within a host machine. If your host has multiple
-storage drives, you may need to remove one ``ceph-osd`` daemon for each drive.
-Generally, it's a good idea to check the capacity of your cluster to see if you
-are reaching the upper end of its capacity. Ensure that when you remove an OSD
-that your cluster is not at its ``near full`` ratio.
+It is possible to remove an OSD manually while the cluster is running: you
```

Comment:
s/while the cluster is running/

Reply:
This is the rare case where I disagree. I think that this "while the cluster is running" clause helps the ignorant reader.